ui/src/utils/openUrl.ts

```diff
@@ -3,13 +3,8 @@
  *  Licensed under the MIT License. See License.md in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-// tslint:disable-next-line:no-require-imports
-import opn = require("opn");
+import * as vscode from 'vscode';
 
 export async function openUrl(url: string): Promise<void> {
-    // Using this functionality is blocked by https://github.com/Microsoft/vscode/issues/25852:
-    // await vscode.commands.executeCommand('vscode.open', vscode.Uri.parse(url));
-
-    // tslint:disable-next-line: no-unsafe-any
-    opn(url);
```

Comment:
> I've been doing some work to eliminate warnings/errors/etc. throughout our build process; one of them is that opn is deprecated.

If this is your only goal, pretty sure you can just switch us to `open` to fix the warning. Should also update this, though: https://github.com/microsoft/vscode-azuretools/blob/main/dev/src/webpack/getDefaultWebpackConfig.ts#L26

Reply:
Still need the default webpack config updated

Reply:
Should I do vscode-azureextensiondev first then? i.e. as a separate PR?

Reply:
#869